mabwiser/neighbors.py

```diff
@@ -75,12 +75,9 @@ def predict_expectations(self, contexts: np.ndarray = None) -> Union[Dict[Arm, N
         return self._parallel_predict(contexts, is_predict=False)
 
     def warm_start(self, arm_to_features: Dict[Arm, List[Num]], distance_quantile: float):
-        # Can only execute warm start when learning policy has been fit in _get_nhood_predictions
```

Comment:
It seems we had "some" definition of what WS is when NP is used? Any idea what it was?

Reply:
Yes we made an attempt. Basically, for Neighbors it would warm start the LP that is trained on all the neighbor contexts. Note that this happens during predict so we cannot really say if the MAB has any cold arms or not since it will depend on the context being predicted. We had something similar for Clusters, the implementation details differ.